cropimageview/build.gradle

```diff
@@ -1,14 +1,22 @@
 apply plugin: 'com.android.library'
-apply plugin: 'com.github.dcendents.android-maven'
-apply plugin: 'com.jfrog.bintray'
+apply plugin: 'com.novoda.bintray-release'
+
+buildscript {
+  repositories {
+    jcenter( )
+  }
+  dependencies {
+    classpath 'com.novoda:bintray-release:0.3.4'
+  }
+}
 
 android {
-  compileSdkVersion 21
-  buildToolsVersion "21.1.2"
+  compileSdkVersion 23
+  buildToolsVersion "22.0.1"
 
   defaultConfig {
-    minSdkVersion 14
-    targetSdkVersion 21
+    minSdkVersion 7
```

Comment:
yep. But I had Dev broken... so I had to do some more changes...

Shame on me

![Uploading sorry mistake.gif…]()